### PR TITLE
Fix Tensor __radd__ type hint issue

### LINKED ISSUE #35213: Tensor __radd__ type hint issue
## 🐛 Bug

Type checking any code adding a float or integer to a tensor (`__radd__` method) results in a type checking error when using nightly build (but not in release). This is causing type checking failures for downstream applications, particularly in Captum.

## To Reproduce

Run mypy on any code applying radd, such as:
```
import torch
ten = torch.tensor([1.0, 2.0, 3.0])
print(7 + ten)
```
Mypy Error:
`test_file.py:4: error: Unsupported operand types for + ("int" and "Tensor")
`

## Expected behavior
No mypy error should occur, as long as the `__radd__` method is appropriately type hinted.

## Additional context

It seems like the problem is caused by a missing comma in this PR: #34552
In tools/pyi/gen_pyi.pi, there should be a comma after 'floor_divide', right now, without a comma, it is concatenating the string with 'radd' and just generating a type hint for an invalid method 'floor_divider_add'

cc @ezyang

### PR DESCRIPTION
**Pull request by pbelevich**
Fixes #35213

Test plan: `mypy -c "import torch; ten = torch.tensor([1.0, 2.0, 3.0]); print(7 + ten)"` should not produce any warnings

Stack from [ghstack](https://github.com/ezyang/ghstack):
* #35167 Move normal() to DistributionTemplates
* **#35231 Fix Tensor __radd__ type hint issue**

Differential Revision: [D20604924](https://our.internmc.facebook.com/intern/diff/D20604924)

### CODE REVIEW

**Review by mruberry — APPROVED** [review]
Awesome!

**Comment by ezyang** [comment]
Thanks!

File: tools/pyi/gen_pyi.py [hunk 1]
```diff
@@ -179,7 +179,7 @@ def arg_to_type_hint(arg):
 
 
 binary_ops = ('add', 'sub', 'mul', 'div', 'pow', 'lshift', 'rshift', 'mod', 'truediv',
-              'matmul', 'floordiv', 'floor_divide'
+              'matmul', 'floordiv', 'floor_divide',
```
Comment (vivekmig) [inline tools/pyi/gen_pyi.py]: @mruberry I think this generating a type hint for "__floor_divide__" but floor_divide doesn't seem to be a supported Python magic method, is that intended?

Comment (mruberry) [reply tools/pyi/gen_pyi.py]: I think your right. I'll look at removing it. Thanks @vivekmig!

**Pull request closed without merging**